filebeat/docs/configuration.asciidoc

```diff
@@ -307,6 +307,15 @@ filebeat:
 -------------------------------------------------------------------------------------
 
 
+===== publish_async
+
+Experimental!
```

Comment:
I don't know of one, but I'll ask those who are wiser in the ways of asciidoc. @debadair or @palecur Do you know if there is a tag that we could use here for experimental config options? Do we have a convention for documenting experimental options?

@urso By experimental, I assume we mean that customers can try the option, but their mileage may vary (and there's no guarantee that the option will be supported in future versions). If so, maybe we need a sentence that basically conveys "use at your own risk."